write-config.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
add `set -euo pipefail` to enforce your bashcript

docs: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
I don't want to. It would mean I'd have to modify the script completely.

Reply:
Using `set -u`. The rest would have required a complete rewrite.